modules/nw-egress-router-about.adoc

```diff
@@ -37,7 +37,12 @@ The egress router image is not compatible with Amazon AWS, Azure Cloud, or any o
 [id="nw-egress-router-about-modes_{context}"]
 == Egress router modes
 
-In _redirect mode_, an egress router pod configures `iptables` rules to redirect traffic from its own IP address to one or more destination IP addresses. Client pods that need to use the reserved source IP address must be modified to connect to the egress router rather than connecting directly to the destination IP.
+In _redirect mode_, an egress router pod configures `iptables` rules to redirect traffic from its own IP address to one or more destination IP addresses. Client pods that need to use the reserved source IP address must be configured to access the service for the egress router rather than connecting directly to the destination IP. The following example demonstrates how to access the destination service and port from the application pod:
```

Comment:
The supplementary style guide states that we should "avoid self-referential language" in writing short descriptions, and I think this also has a touch of anthropomorphism ("...example demonstrates...")

Suggestion (phrased to avoid sounding like a procedure step): 
```
You can access the destination service and port from the application pod by using the `curl` command. For example:
```

Reply:
Thanks, @ousleyp . I never thought of how I structured the sentence in this manner. Thanks for pointing this out. I've update the sentence according to your recommendation.